UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/MetaDataNode.cs

```diff
@@ -164,8 +169,9 @@ public void TryCreateDamageInfo(LayerType layerType)
 	/// Create a new MetaDataNode on the specified local position (within the parent matrix)
 	/// </summary>
 	/// <param name="position">local position (within the matrix) the node exists on</param>
-	public MetaDataNode(Vector3Int position, ReactionManager reactionManager, Matrix matrix)
+	public MetaDataNode(Vector3Int position, ReactionManager reactionManager, Matrix matrix, MetaDataSystem InMetaDataSystem )
```

Comment:
Couldn't you have got it from the reaction manager?

Reply:
nah, the only reference to it is a SubsystemBehaviour, as part of a list